src/backend/txn.go

```diff
@@ -307,7 +307,9 @@ func (txn *Txn) Commit() error {
 }
 
 // Rollback used to rollback a XA transaction.
-// 1. XA ROLLBACK
+// 1. XA END
+// 2. XA PREPARE
+// 3. XA ROLLBACK
```

Comment:
Here we can keep consistent with func RollbackScatter(), delete the comment or add comment 
```
// 1. XA END
// 2. XA PREPARE
// 3. XA ROLLBACK
```
before RollbackScatter() too.